projects/Mallard/src/components/sportscore/sportscore.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react'
+import { View, Text, StyleSheet, StyleProp, ViewStyle } from 'react-native'
+import { metrics } from 'src/theme/spacing'
+import { getFont } from 'src/theme/typography'
+import { color } from 'src/theme/color'
+
+const styles = StyleSheet.create({
+    background: {
+        backgroundColor: color.palette.highlight.main,
+        padding: 0,
+        paddingBottom: 2,
+        paddingHorizontal: metrics.horizontal / 3,
+    },
+    text: {
+        letterSpacing: 0.25,
+        ...getFont('headline', 0.5),
+    },
+})
+
+const SportScore = ({
+    style,
```

Comment:
Something to discuss at a later date, but i dont feel its good practice to keep passing style through as a prop. Leads to "incomplete components"